includes/class-sensei-settings.php

```diff
@@ -204,7 +204,7 @@ public function init_fields() {
 
 		$fields['access_permission'] = array(
 			'name'        => __( 'Access Permissions', 'woothemes-sensei' ),
-			'description' => __( 'Users must be logged in to view Course and Lesson content.', 'woothemes-sensei' ),
+			'description' => __( 'Users must be logged in to view Lesson content.', 'woothemes-sensei' ),
```

Comment:
Could we also change "Lesson" so its not capitalized here? The case is inconsistent throughout Sensei and we should just pick one. My vote is for lower case.

Reply:
Done in 91d85bbe2c76efe7795c5026572732b126c70bad